build-test.cmd

```diff
@@ -191,7 +190,7 @@ if defined __UpdateInvalidPackagesArg (
   set __up=-updateinvalidpackageversions
 )
 
-call %~dp0run.cmd build -buildManagedTests -MsBuildLog=!__msbuildLog! -MsBuildWrn=!__msbuildWrn! -MsBuildErr=!__msbuildErr! %__up% %__RunArgs% %__unprocessedBuildArgs%
+call %__ProjectDir%\run.cmd build -buildManagedTests -MsBuildLog=!__msbuildLog! -MsBuildWrn=!__msbuildWrn! -MsBuildErr=!__msbuildErr! %__up% %__RunArgs% %__unprocessedBuildArgs%
 if errorlevel 1 (
```

Comment:
The -buildmanagedTests uses relative paths in config.json. Replace this with ```-Project=%__ProjectDir%/tests/build.proj``` and remove the -buildManagedTests flag from ```config.json```